### ISSUE TITLE
Missing Facebook share.php dialog

### ISSUE THREAD

**Issue by maxtherocket** [vweevers/share-dialog#3]
The only feature that is missing in the module is the Facebook share.php dialog, that simply opens a new popup window with the following URL http://www.facebook.com/sharer/sharer.php?u=$URL

I would love to see this dialog added to the module :)

**Comment by njam3** [comment]
Agreed, I would love to see this added. I use this more often then the facebook sdk share.

**Comment by vweevers** [comment]
I left that one out because it was deprecated. It seems that's no longer true (because everybody kept using it?). So OK, I will add it.

As for how to differentiate between the two: should I simply change the signature of `Dialog.facebook` so that `app_id` and `redirect` become optional?

``` js
// Would return https://facebook.com/dialog/share?xx
Dialog.facebook(app_id, href, redirect_uri) 

 // Would return https://facebook.com/sharer/sharer.php?xx
Dialog.facebook(href)
```

That would work unless Facebook has also "undeprecated" the other params (besides `u`). Do you have info on that?

**Comment by njam3** [comment]
I think that method would work well. As far as I know, "u" is the only variable you can pass to share.php as it picks up all the other data from the open graph tags on the page.

**Comment by vweevers** [comment]
Added in 18d2dd7, released 0.2.0.
